src/common-ssh/ssh.c

```diff
@@ -479,7 +478,7 @@ guac_common_ssh_session* guac_common_ssh_create_session(guac_client* client,
             guac_client_log(client, GUAC_LOG_DEBUG, "Unable to connect to "
                     "host %s, port %s: %s",
                     connected_address, connected_port, strerror(errno));
-
+        close(fd);
```

Comment:
Previously, there was a blank line after this single-line `else` block, visually separating that block from the code that followed. For the sake of readability, that blank line should remain.

I would suggest either restoring the blank line, adding this `close()` call to the `else` block itself, or getting rid of the `else` entirely and rely on the fact that the `if` will `break` out of the loop (and thus the `else` is superfluous anyway).